apps/indexer/lib/indexer/fetcher/celo_epoch_data.ex

```diff
@@ -152,6 +152,13 @@ defmodule Indexer.Fetcher.CeloEpochData do
     Map.merge(block, %{voter_rewards: voter_rewards})
   end
 
+  def calculate_voter_rewards(after_rewards_votes, before_rewards_votes, _plus_revoked_minus_activated_votes)
+      when after_rewards_votes == before_rewards_votes,
```

Comment:
Isn't this case handled by [this line](https://github.com/celo-org/blockscout/pull/814/files#diff-40d04a4c903621dde0aeebe53400212cc4327b1a244396f05ad7469e87aa1091R163)?